src/node_v8.cc

```diff
@@ -151,6 +177,35 @@ void Initialize(Local<Object> target,
   HEAP_STATISTICS_PROPERTIES(V)
 #undef V
 
+  // Export symbols used by v8.getHeapCodeStatistics()
+  env->SetMethod(target,
+                 "updateHeapCodeStatisticsArrayBuffer",
+                 UpdateHeapCodeStatisticsArrayBuffer);
+
+  env->set_heap_code_statistics_buffer(
+    new double[kHeapCodeStatisticsPropertiesCount]);
```

Comment:
I think this is missing a corresponding `delete[]` call in env.cc?

Reply:
fixed